Sources/FileSystem/FileSystem.swift

```diff
@@ -549,6 +549,14 @@ public struct FileSystem: FileSysteming, Sendable {
         )
     }
 
+    public func createSymbolicLink(from: AbsolutePath, to: RelativePath) async throws {
+        logger?.debug("Creating symbolic link from \(from.pathString) to \(to.pathString).")
+        try await NIOFileSystem.FileSystem.shared.createSymbolicLink(
+            at: FilePath(from.pathString),
+            withDestination: FilePath(to.pathString)
+        )
```

Comment:
What do you think about creating a private helper method to combine the two `createSymbolicLink` methods?
```swift
    private func createSymbolicLink(fromPathString: String, toPathString: String) async throws {
        logger?.debug("Creating symbolic link from \(fromPathString) to \(toPathString).")
        try await NIOFileSystem.FileSystem.shared.createSymbolicLink(
            at: FilePath(fromPathString),
            withDestination: FilePath(toPathString)
        )
    }

    public func createSymbolicLink(from: AbsolutePath, to: AbsolutePath) async throws {
        try await createSymbolicLink(fromPathString: from.pathString, toPathString: to.pathString)
    }
```